mitosheet/mitosheet/public/v3/sheet_functions/number_functions.py

```diff
@@ -103,9 +302,129 @@ def MULTIPLY(*argv: NumberInputType) -> NumberFunctionReturnType:
+        "search_terms": ["power", "raise", "exponent", "square", "cube"],
+        "examples": [
+            "POWER(4, 1/2)",
+            "POWER(Dose, 2)"
+        ],
+        "syntax": "POWER(value, exponent)",
+        "syntax_elements": [{
+                "element": "value",
+                "description": "Number to raise to a power."
+            },
+            {
+                "element": "exponent",
+                "description": "The number to raise value to."
+            }
+        ]
+    }
+    """
+    if isinstance(series, int) or isinstance(series, float):
+        return series ** power
+    
+    return series.pow(power) # type: ignore
+
+@cast_values_in_arg_to_type('series', 'number')
+@cast_values_in_arg_to_type('decimals', 'number')
+@handle_sheet_function_errors
+def ROUND(series: NumberRestrictedInputType, decimals: Optional[IntRestrictedInputType]=None) -> NumberFunctionReturnType:
+    """
+    {
+        "function": "ROUND",
+        "description": "Rounds a number to a given number of decimals.",
+        "search_terms": ["round", "decimal", "integer"],
+        "examples": [
+            "ROUND(1.3)",
+            "ROUND(A, 2)"
+        ],
+        "syntax": "ROUND(value, [decimals])",
+        "syntax_elements": [{
+                "element": "value",
+                "description": "The value or series to round."
+            },
+            {
+                "element": "decimals",
+                "description": " The number of decimals to round to. Default is 0."
+            }
+        ]
+    }
+    """
+
+    # If no decimals option is passed, round to no decimals
+    if decimals is None:
+        decimals = 0
+
+    index = get_index_from_series(series, decimals)
+    series = get_series_from_primitive_or_series(series, index)
+    decimals = get_series_from_primitive_or_series(decimals, index)
+
+    return pd.Series(
+        [round(num, dec) for num, dec in zip(series, decimals)], # type: ignore
+        index=series.index # type: ignore
+    )
+
+@cast_values_in_arg_to_type('series', 'number')
+@handle_sheet_function_errors
+def SKEW(series: NumberRestrictedInputType) -> NumberFunctionReturnType:
+    """
+    {
+        "function": "SKEW",
+        "description": "Computes the skew of a series, excluding missing values.",
+        "search_terms": [],
+        "examples": [
+            "=SKEW(A)",
+            "=SKEW(A * B)"
+        ],
+        "syntax": "SKEW(series)",
+        "syntax_elements": [{
+                "element": "series",
+                "description": "The series to calculate the skew of."
+            }
+        ]
+    }
+    """
+    if isinstance(series, int) or isinstance(series, float):
+        return 0
+        
+    return series.skew() # type: ignore
+
+@cast_values_in_arg_to_type('series', 'number')
+@handle_sheet_function_errors
+def STDEV(series: NumberRestrictedInputType) -> NumberFunctionReturnType:
```

Comment:
According to our spec, `STDEV` should also support dataframes and rolling ranges.